scripts/build/docker/Dockerfile-i686

```diff
@@ -1,8 +1,15 @@
 FROM erwinchang/ubuntu-12.04-32bit-build
 
+# Setup APT sources
+
+RUN sed s,ubuntu\.stu\.edu\.tw,archive.ubuntu.com, /etc/apt/sources.list > /tmp/sources.list \
+  && mv /tmp/sources.list /etc/apt/sources.list
+
+
+
+
```

Comment:
Can we trim the excessive white-space after a block and between comment & block?